Simplified/Announcements/NYPLAnnouncementBusinessLogic.swift

```diff
@@ -69,6 +74,45 @@ class NYPLAnnouncementBusinessLogic {
                                           "presentedAnnouncements": presentedAnnouncements])
     }
   }
+  
+  // MARK: - Helper
+  
+  /**
+  Generates an alert view that presents another alert when being dismissed
+  - Parameter announcements: an array of announcements that goes into alert message.
+  - Returns: The alert controller to be presented.
+  */
+  private func alert(announcements: [Announcement]) -> UIAlertController? {
```

Comment:
Moved this function as part of NYPLAnnouncementBusinessLogic as @ettore suggest, since it is only being called by the business logic and to avoid tangling.